### PR TITLE
Fixing v2 actions-octopus

### PR DESCRIPTION
**Pull request by vibindaniel**
# Description

Refactored v2 and added terraform compatibility

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Tested in demo-app repository. Both helm, terraform & exception worked

- [ ] Sanity Testing

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: replicator/replicator.ps1 [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+$TerraformDir = "$env:GITHUB_WORKSPACE/$env:TERRAFORM_DIR_PATH"
+
+if (Test-Path -Path "$TerraformDir/*.tf")
+{
+  Write-Information "Terraform found at $TerraformDir"
+  $IsPresent = $true
+  Invoke-EpsTemplate -Path $PSScriptRoot/templates/terraform_dynamodb.eps `
+  | Out-File -FilePath $TerraformDir/iaac_replicator_dynamodb.tf
+}
```
Comment (naveen-variant) [inline replicator/replicator.ps1:11]: Where are you running terraform apply to create entries in dynamo db able.

Comment (vibindaniel) [reply replicator/replicator.ps1:11]: Same place where helm is run

Comment (naveen-variant) [reply replicator/replicator.ps1:11]: Do you mean as part of deploy.ps1, thats where we run helm commands.

Comment (vibindaniel) [reply replicator/replicator.ps1:11]: Depends on the entrypoint in a project

Comment (naveen-variant) [reply replicator/replicator.ps1:11]: got it.

Comment (vibindaniel) [reply replicator/replicator.ps1:11]: https://octopus.apps.ops-drivevariant.com/app#/Spaces-2/projects/replicator/deployments/releases/0.1.0-add-octopus-v2-0001.319/deployments/Deployments-9239?tasklineid=0%2FServerTasks-50834_QNXHJYSJRL%2F1cec407d68664f27b4c1976f4d9d6c88%2F6bd8ddc7839a4fc18d486577350e2fd3&activeTab=taskLog

**Review by naveen-variant — APPROVED** [review]

**Pull request merged**